cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,17 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+# In the following a volume is created to contain the nodered config, this is done to
+# allow starting the nodered container from inside a devcontainer
+
+# Create docker volume to contain nodered config
+docker volume create nodered-config-volume
+
+# Create temporarily container to copy nodered config into the created volume
+docker create -v nodered-config-volume:/data --name nodered-config-container debian:12-slim true
+
+# Copy nodered config to the created volume with the temporarily created container
+docker cp @FLOW_FILE@ nodered-config-container:/data/flows.json
```

Comment:
`@FLOW_FILE@` seems to be the only parameter used in this template. Wouldn't it be possible to just use it as a regular script, which gets the flow file passed as an argument?

Reply:
Would be possible, but as far as i know this run nodered script was introduced to provide a convenient way to start nodered with correct flow files. When having a script with parameter you would need to know about the path of this flow file and type it..

Reply:
yeah, that's a good point